long-term-storage.md

```diff
@@ -110,6 +109,8 @@ $ helm install kubecost/cost-analyzer \
 
 Your deployment should now have Thanos enabled!
 
+> Note: if you delete a previous install of kubecost, do this before creating the secret in step 2
```

Comment:
How about "if you *need* to delete an existing non-Enterprise install"... something to give context on why this would be neeed.